providers-sdk/v1/plugin/grpc.go

```diff
@@ -87,6 +87,10 @@ func (m *GRPCServer) Connect(ctx context.Context, req *ConnectReq) (*ConnectRes,
 	return m.Impl.Connect(req, a)
 }
 
+func (m *GRPCServer) Shutdown(ctx context.Context, req *ShutdownReq) (*ShutdownRes, error) {
```

Comment:
Does that get called anywhere? Seems it's unused in this PR

Reply:
yes, the calling part was implemented in #1632 but wasn't test with non-builtin providers. This code was missing so for all installed provider a not implemented error was returned